### PR TITLE
Added SAP's Hacktoberfest Swag 2019

### PR DESCRIPTION
**Pull request by CoderJolly**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

I have added the SAP Hacktoberfest Swag List 2019 #371

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -277,6 +277,15 @@
     "image": "https://i.imgur.com/gVfuWsu.png",
     "dateAdded": "2018-10-20T11:38:34.000Z",
     "tags": ["clothing", "accessories", "expired", "stickers"]
+  },
+   {
+    "name": "SAP",
+    "difficulty": "medium",
+    "description": "Make a valid change in one abap Project and submit PR",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "description": "Submit a valid PR to an ABAP Project",
```

File: data.json [hunk 2]
```diff
@@ -277,6 +277,15 @@
...
+    "reference": "https://blogs.sap.com/2019/10/04/lets-code-share-and-learn.-calling-abapers-to-next-edition-of-hacktoberfest/",
+    "image": "https://blogs.sap.com/wp-content/uploads/2019/10/hacktoberfest.jpg",
+    "dateAdded": "2019-10-05T13:40:00.000Z",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "dateAdded": "2019-10-07T20:03:00.000Z",
```

**Comment by CoderJolly** [comment]
@vikaspotluri123 Could you pls let me know why the Travis test is failing ?

File: data.json [hunk 3]
```diff
@@ -277,6 +277,15 @@
     "image": "https://i.imgur.com/gVfuWsu.png",
     "dateAdded": "2018-10-20T11:38:34.000Z",
     "tags": ["clothing", "accessories", "expired", "stickers"]
+  },
+   {
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
  {
```

**Comment by aslafy-z** [comment]
Let's add the `hacktoberfest` tag ;)

File: data.json [hunk 4]
```diff
@@ -278,6 +278,15 @@
...
     "tags": ["clothing", "accessories", "expired", "stickers"]
   },
+  {
+    "name": "SAP",
+    "difficulty": "medium",
+    "description": "Submit a valid PR to an ABAP Project",
+    "reference": "https://blogs.sap.com/2019/10/04/lets-code-share-and-learn.-calling-abapers-to-next-edition-of-hacktoberfest/",
+    "image": "https://blogs.sap.com/wp-content/uploads/2019/10/hacktoberfest.jpg",
+    "dateAdded": "2019-10-07T20:03:00.000Z",
+    "tags": ["accessories"]
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "tags": ["accessories", "hacktoberfest"]
```

File: data.json [hunk 5]
```diff
@@ -278,6 +278,15 @@
     "dateAdded": "2018-10-20T11:38:34.000Z",
     "tags": ["clothing", "accessories", "expired", "stickers"]
   },
+  {
+    "name": "SAP",
+    "difficulty": "medium",
+    "description": "Submit a valid PR to an ABAP Project",
```
Comment (aslafy-z) [inline data.json]: And then what? Can you develop the `valid` term? Thanks!

**Comment by aslafy-z** [comment]
Sorry for that one. Thanks @CoderJolly

**Pull request closed without merging**